libs/template/labs/src/articles/article-routes.ts

```diff
@@ -1,32 +1,22 @@
 import { html, TemplateResult } from 'lit';
+import { ContentfulContentFields } from './contentful';
+import { StoryblokContentFields } from './storyblok';
 
 export const articleRoutes = [
-  {
-    path: '/faq',
-    render: (): TemplateResult => html`<oryx-composition
-      route="/faq"
-    ></oryx-composition>`,
-    name: 'FAQs',
-  },
   {
     path: '/faq/:id',
     render: (): TemplateResult => html`<oryx-composition
       route="/faq/:id"
     ></oryx-composition>`,
     name: 'FAQ',
-  },
-  {
-    path: '/article',
-    render: (): TemplateResult => html`<oryx-composition
-      route="/article"
-    ></oryx-composition>`,
-    name: 'Articles',
+    type: StoryblokContentFields.Faq,
```

Comment:
This route config is an example where we'd hope that the render is not needed, as we can have a fallback using the `path`. Optionally, however, one could bring in a custom render method.

Reply:
But without router definition we don't know if we have this route at all